### ISSUE TITLE
linux/arm64 support for golangci-lint is missing

### ISSUE THREAD

**Issue by mmihira** [williamboman/mason.nvim#1084]
### I've searched open issues for similar requests

Yes

### I've manually reviewed logs to find potential errors

Yes

### I've recently downloaded the latest plugin version of mason.nvim

- [x] Yes

### Problem description

I'm running `MasonInstall golangci-lint` in a `linux/arm64` docker environment. The command fails. 

### Expected behavior

This is not working because this platform combination is not supported by this package even though golagnci-lint does support this platform. 

I think we just have to add the option here: `lua/mason-registry/golangci-lint/init.lua`
..

`when(platform.is.linux_arm64, release_file "golangci-lint-%s-linux-armv64.tar.gz")`

### Steps to reproduce

`MasonInstall golangci-lint`

### Affected packages

goalngci-lint

### Neovim version (>= 0.7)

>= 0.7

### Operating system/version

linux/arm64

### Healthcheck

```Text
Mason is fine.
```

**Comment by williamboman** [comment]
PR welcomed!On 9 Mar 2023, at 14:17, mmihira ***@***.***> wrote:﻿
I've searched open issues for similar requests
Yes
I've manually reviewed logs to find potential errors
Yes
I've recently downloaded the latest plugin version of mason.nvim

 Yes

Problem description
I'm running MasonInstall golangci-lint in a linux/arm64 docker environment. The command fails.
Expected behavior
This is not working because this platform combination is not supported by this package even though golagnci-lint does support this platform.
I think we just have to add the option here: lua/mason-registry/golangci-lint/init.lua
..
when(platform.is.linux_arm64, release_file "golangci-lint-%s-linux-armv64.tar.gz")
Steps to reproduce
MasonInstall golangci-lint
Affected packages
goalngci-lint
Neovim version (>= 0.7)

= 0.7

Operating system/version
linux/arm64
Healthcheck
Mason is fine.

Screenshots or recordings
No response
